### DISCUSSION TITLE
GnuPG card list stalls while Solo2 is connected

### DISCUSSION THREAD

**Discussion by schlomie** [solokeys/solo2#141] [General]
[This](https://github.com/Nitrokey/nitrokey-3-firmware/issues/22) is happening when Solo2 is connected too.  Since the NK3 and Solo2 share similar firmware, I imagine the problem and fix are also very similar.

**Comment by baldurmen** [comment]
Thanks for the debugging. I've hit the same issue and fixed it this way:

https://github.com/solokeys/solo2-cli/issues/107#issuecomment-1595655657
